src/components/IconLibrary/icon-library.scss

```
@@ -47,53 +77,122 @@
   }
 }
 
-.icon-search {
-  margin-bottom: rem(32px);
+.icons-list {
+  display: flex;
+  flex-flow: wrap;
+  width: 100%;
+  padding-bottom: $spacing-06;
 }
 
-.icon {
+.icon__container {
+  flex: 0 1 50%;
+  position: relative;
+
   &:nth-child(n) {
-    border-left: 1px solid $carbon--gray-20;
     border-top: 1px solid $carbon--gray-20;
+    border-left: 1px solid $carbon--gray-20;
   }
-
-  &:nth-child(-n + 4) {
+  &:nth-child(n):nth-child(-n + 2) {
     border-top: none;
   }
-
-  &:nth-child(4n + 1) {
+  &:nth-child(2n - 1) {
     border-left: none;
   }
 
-  @include carbon--breakpoint('lg') {
+  & .bx--aspect-ratio > * {
```

```suggestion
  .bx--aspect-ratio > * {
```

this is not needed right?